unit2/unit2.ipynb

```diff
@@ -1030,9 +1031,8 @@
         "  api = HfApi()\n",
         "  \n",
         "  repo_url = api.create_repo(\n",
-        "        name=repo_name,\n",
+        "        repo_id=repo_id,\n",
         "        token=token,\n",
-        "        organization=organization,\n",
```

Comment:
This should get rid of the warnings that popup when tyring to call `push_to_hub`.

Reply:
Thanks! Could you please remove the line `organization, repo_name = repo_id.split("/")`? It will not be needed anymore

Reply:
Also `huggingface_token = HfFolder.get_token()` is not really needed, so you can remove that

Reply:
> Thanks! Could you please remove the line organization, repo_name = repo_id.split("/")? It will not be needed anymore

Actually realized we do need `repo_name` few lines below so I've added the below line.

```python
  _, repo_name = repo_id.split("/")
```